buildbot/master/files/config/environments.py

```diff
@@ -85,8 +99,9 @@ def __add__(self, other):
     'PKG_CONFIG_ALLOW_CROSS': '1',
 })
 
-
-build_arm32 = build_arm + Environment({
+# Unset SERVO_CACHE_DIR to test our download code for host and cross targets.
+# Use arm32 because it is the fastest cross builder.
+build_arm32 = build_arm.without(['SERVO_CACHE_DIR']) + Environment({
```

Comment:
I unfortunately don't think this will work. All that `SERVO_CACHE_DIR` does is allow us to share the cached bits between different builds that run on the same slave. I think that in order to do this we will have to either:
1) Clear out the .servo directory entirely on each run
2) Add a step that does a `./mach boostrap-rust --force`